code/datums/elements/climbable.dm

```diff
@@ -31,11 +34,13 @@
 
 /datum/element/climbable/proc/on_examine(atom/source, mob/user, list/examine_texts)
 	SIGNAL_HANDLER
-
-	if(can_climb(source, user))
-		examine_texts += span_notice("[source] looks climbable.")
+	examine_texts += span_notice("[source] looks climbable.")
 
 /datum/element/climbable/proc/can_climb(atom/source, mob/user)
+	var/dir_step = get_dir(user,source.loc)
+	//To jump over a railing you have to be standing next to it, not far behind it.
+	if(on_border && user.loc != source.loc && (dir_step & source.dir) == source.dir)
```

Comment:
Can't you check if the `source` has the on border flag instead of adding a new parameter and variable for it?

Reply:
This is not TGMC or baystation, I believe we don't have a on_border flag for atoms yet.

Reply:
We do. This should cover it:
```dm
	if(flags_1 & ON_BORDER_1)
```

Reply:
Oh, ON_BORDER**_1**, that's why...
I'll go and change railings then, because they currently don't make use of that flag.